__tests__/main.test.ts

```diff
@@ -0,0 +1,116 @@
+import { run } from "../src/labeler";
+import { GitHub } from "@actions/github";
+import * as core from "@actions/core";
+
+const fs = jest.requireActual("fs");
+
+jest.mock("@actions/core");
+jest.mock("@actions/github");
+
+const gh = new GitHub("_");
+const addLabelsMock = jest.spyOn(gh.issues, "addLabels");
+const removeLabelMock = jest.spyOn(gh.issues, "removeLabel");
+const reposMock = jest.spyOn(gh.repos, "getContents");
+const paginateMock = jest.spyOn(gh, "paginate");
+const getPullMock = jest.spyOn(gh.pulls, "get");
+
+const yamlFixtures = {
+  "only_pdfs.yml": fs.readFileSync("__tests__/fixtures/only_pdfs.yml"),
+};
+
+function usingLabelerConfigYaml(fixtureName: keyof typeof yamlFixtures): void {
```

Comment:
nit: for convention I would move these helper functions to the bottom of the file

Reply:
Wasy enough! Will do. 👍